initfiles/componentfiles/configxml/thor.xsd.in

```diff
@@ -462,14 +462,14 @@
           </xs:appinfo>
         </xs:annotation>
       </xs:attribute>
-      <xs:attribute name="masterport" type="xs:nonNegativeInteger" default="20000">
+      <xs:attribute name="masterport" type="xs:nonNegativeInteger" default="20000" use="optional">
```

Comment:
I thought there was a recent unrelated change to make this not optional so that it was always present in the xml for nagios and other external programs to be able to determine the thormaster port to scan.